Source/WebCore/inspector/InspectorInstrumentation.cpp

```diff
@@ -752,12 +752,12 @@ void InspectorInstrumentation::didCommitLoadImpl(InstrumentingAgents& instrument
     ASSERT(loader->frame() == &frame);
 
     if (frame.isMainFrame()) {
-        if (auto* consoleAgent = instrumentingAgents.webConsoleAgent())
-            consoleAgent->reset();
-
         if (auto* networkAgent = instrumentingAgents.enabledNetworkAgent())
             networkAgent->mainFrameNavigated(*loader);
 
+        if (auto* consoleAgent = instrumentingAgents.webConsoleAgent())
+            consoleAgent->mainFrameNavigated();
```

Comment:
does the ordering of this actually matter?  if so we should either add a comment or figure out a way to put that into the code so it's not changed in the future (and write a test for it too)